pkg/controller/initializer.go

```diff
@@ -13,6 +14,7 @@ func init() {
 	addManager := func(mgr manager.Manager) error {
 		k8sService := GetInstance(mgr)
 		reconciler := kieapp.Reconciler{Service: &k8sService}
+		webconsole.LoadWebConsoleEnrichmentUI()
```

Comment:
It seems to me that this is similar to creating our operator UI. We do that [here](https://github.com/kiegroup/kie-cloud-operator/blob/master/pkg/controller/kieapp/kieapp_controller.go#L58-L60), so I would say this line would go right next to it. @tchughesiv , wdyt?

Reply:
moved. 
it looks like a proper place to put the yaml sample creation method call.